### ISSUE TITLE
Global 1/4 deg static B is broken

### ISSUE THREAD

**Issue by guillaumevernieres** [JCSDA-internal/soca#732]
## Description
I just realized yesterday that our static B does not "work" at 1/4 deg resolution, while the same configuration works for 1 deg. 
I'll try to track down more precisely what's going on today, hopefully. 

### To Reproduce:
@benjaminmenetrier , @travissluka , I will provide a scratch of static files and a run scripts that reproduces the error.

I tested both, "SocaError" and "SABER/BUMP_NICAS" and I get the same error.

## Output
This is the log I keep getting when running the var application after a "fresh" initialization of bump:

```console
[...]
-------------------------------------------------------------------
--- Read local NICAS parameters
       Read NICAS local data
          Read NICAS data of task      1
             Group var-var
!!! NetCDF error in netcdf_inquire_var on task #000011: NetCDF: Variable not found
!!! Hint: variable inorm
       |-> var-var
          |-> ocn_nicas
Abort(1) on node 10 (rank 10 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 1) - process 10
[...]
```
I'm pretty sure we initialize bump properly, with the correct variables. Not sure why it is looking for ``inorm``. The same configuration works fine at 1 deg resolution.

**Comment by guillaumevernieres** [comment]
@benjaminmenetrier , @travissluka After testing with the dirac and var application and writing the yamls from scratch instead of using the ones generated within ``soca-science``, it turns out all is good. The bug is in our yaml templates ... somewhere.

**Comment by guillaumevernieres** [comment]
@benjaminmenetrier , @travissluka I'm re-opening the issue. False good news. 
When I re-wrote the yamls, I also reduced the number of pe's I was testing with, something about being impatient waiting in the queue a minute instead of 2s ... It works with 100, 200 but 480 gives the error above.

Done working on this for now. I'll find a layout that works and use that in the meantime.

I got curious about the look of the decomposition, thinking maybe something weird was happening. Doesn't look like it:

![decomp-480](https://user-images.githubusercontent.com/14031856/161328785-f2eed702-b25c-43fa-a491-45547e5dbcbb.png)

Each rectangle is a pe domain. I can't see anything obviously wrong.

**Comment by guillaumevernieres** [comment]
Note to self: 512pe works.

**Comment by benjaminmenetrier** [comment]
The field `inorm` is written ([here](https://github.com/JCSDA-internal/saber/blob/39cf4df2cd83ca5e5043d153f618074ac59a6f10/src/saber/bump/type_nicas_blk.fypp#L857)) only if the number of points of the NICAS subgrid is > 0 for this PE. It is then read ([here](https://github.com/JCSDA-internal/saber/blob/39cf4df2cd83ca5e5043d153f618074ac59a6f10/src/saber/bump/type_nicas_blk.fypp#L686)) with the same condition.

This expected behavior was buggy, and it was fixed 3 weeks ago with https://github.com/JCSDA-internal/saber/pull/185. Is your branch up-to-date with `develop`?

**Comment by guillaumevernieres** [comment]
> The field `inorm` is written ([here](https://github.com/JCSDA-internal/saber/blob/39cf4df2cd83ca5e5043d153f618074ac59a6f10/src/saber/bump/type_nicas_blk.fypp#L857)) only if the number of points of the NICAS subgrid is > 0 for this PE. It is then read ([here](https://github.com/JCSDA-internal/saber/blob/39cf4df2cd83ca5e5043d153f618074ac59a6f10/src/saber/bump/type_nicas_blk.fypp#L686)) with the same condition.
> 
> This expected behavior was buggy, and it was fixed 3 weeks ago with [JCSDA-internal/saber#185](https://github.com/JCSDA-internal/saber/pull/185). Is your branch up-to-date with `develop`?

That works, thanks @benjaminmenetrier :tada:
